api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1248,9 +1248,72 @@ const (
 // +kubebuilder:validation:Enum=Karpenter
 type Provisioner string
 
+// Configures when and how to scale down cluster nodes.
+type ScaleDownConfig struct {
+	// enabled determines whether the Cluster Autoscaler should scaled down this cluster.
+	Enabled bool `json:"enabled"`
+
+	// delayAfterAdd sets how long after scale up the scale down evaluation resumes
+	// +kubebuilder:validation:Pattern=([0-9]*(\.[0-9]*)?[a-z]+)+
+	// +kubebuilder:validation:MaxLength=100
+	// +optional
+	DelayAfterAdd *string `json:"delayAfterAdd,omitempty"`
```

Comment:
For each of the delays, what is the behaviour if I were to explicitly set `0` as the value?